packages/ui/src/ui/components/ACL/ACL.js

```diff
@@ -429,13 +430,16 @@ class ACL extends Component {
     }
 
     renderColumnGroups() {
-        const {columnGroups, idmKind, path, loadAclData, cluster, aclRequestOptions} = this.props;
+        const {columnGroups, idmKind, path, loadAclData, cluster, aclRequestOptions, nodeType} =
+            this.props;
         const {useEffective} = aclRequestOptions;
+        const allowedNodeTypes = ['map_node'];
         const props = {
             path,
             loadAclDataFn: () => loadAclData({path, idmKind}),
             columnGroups,
             cluster,
+            hasCreateForm: allowedNodeTypes.includes(nodeType),
```

Comment:
At the moment `allowedNodeTypes`  looks like an unnecessary variable:
`nodeType === 'map_node'`